app/Server.hs

```diff
@@ -62,6 +62,7 @@ runServer = do
               dir "get-json-data" $ look "graphName" >>= \graphName -> liftIO $ getGraphJSON graphName,
               dir "loading" $ look "size" >>= loadingResponse,
               dir "save-json" $ look "jsonData" >>= \jsonStr -> look "nameData" >>= \nameStr -> liftIO $ saveGraphJSON jsonStr nameStr,
+              dir "export-pdf" $ exportResponse "HelloWorld",
```

Comment:
So, this week I've been quite confused about how this will all fit together. I've been trying to work it out on my own, but I might just be making this more complicated in my head than it needs to be.

In order to make "this use the same `image` from the image route", should the `export-pdf` route first call the `graphImageResponse` function and then use the resulting image data sent as a response to the `exportResponse`. Following that, would I have to work on finding a way to insert that data into a TEX file which will be converted to a pdf by `exportResponse`? I did some searching and it might be possible to use the filecontents package in Latex to include a base64 encoded PDF in a TEX file.

Or does this mean, that `exportResponse` should retrieve the graph id on the current page in the same way that `graphImageResponse` does, and then use that info to generate a PDF from the SVG which will then somehow be instead into the TEX file?

I actually have a feeling I'm just completely missing the point and the correct approach is something else entirely, but I haven't been able figure it out.